packages/application-shell-connectors/src/components/application-context/application-context.tsx

```diff
@@ -250,6 +250,8 @@ function withApplicationContext<
           const mappedProps = mapApplicationContextToProps
             ? mapApplicationContextToProps(applicationContext)
             : { applicationContext };
+          // @ts-ignore FIXME: this is an issue with HOC typing while using Emotion's JSX namespace
+          // See: https://github.com/emotion-js/emotion/issues/3245, https://github.com/emotion-js/emotion/issues/3261
```

Comment:
This is unrelated but has started causing the type check to fail.
Had to add it in a bunch of places.